generic/tests/cfg/pxe_boot.cfg

```diff
@@ -22,6 +22,9 @@
     no macvtap
     variants:
         - @default:
+            s390x:
+                extra_params += " -no-shutdown"
+                bootindex_nic1 = 0
```

Comment:
Suggest moving this section out of this variant right to the previous level.

Reply:
good idea , updated